### PR TITLE
chore: update build process docs

### PR DESCRIPTION
**Pull request by pfulton**
This updates the Build Process portion of the Standard.

### CODE REVIEW

File: build_process/README.md [hunk 1]
```diff
@@ -7,6 +7,32 @@ outcome.
...
+
+You may hear or see the term "task" or "build task". You can think of a build task as an individual piece of a larger build process. A build task can also serve as a single process that can be executed on its own, outside of the larger build process.
+
+### How we do build processes at Sparkbox
+Our build process is Node-based, meaning that we rely on [Node](https://nodejs.org/en/) to translate our source files into production-ready files. Specifically, our build processes use [npm scripts](https://github.com/sparkbox/standard/blob/master/build_process/node.md#npm-scripts) to handle the translation of source files into build artifacts.
+
+Note that in the past, we have used other Node-based build methods, including [Grunt](https://gruntjs.com/) and [Gulp](https://gulpjs.com/).
+
+Ultimately, regardless of the tools we choose, our goal is to have a build process that fits the projects' needs and our clients' needs, and we strive to strike a balance between having a great developer experience and a great resulting project.
```
Comment (asimpson) [inline build_process/README.md]: :heart:

File: build_process/README.md [hunk 2]
```diff
@@ -1,27 +1,53 @@
...
+
+These are some of our common build tasks:
+* Compiling Sass to CSS
+* Adding browser and vendor prefixes to our CSS by using PostCSS with Autoprefixer
+* Transpiling JavaScript
+* Optimizing and/or minifying code
+
 ## Quality Expectations
 
 There are certain things we expect to happen frequently to help keep quality
```
Comment (nrleak) [inline build_process/README.md]: I'm not sure I quite understand what this means. Is it something along the lines of, 
"We focus on these things to ensure high quality that is obvious to both our team and our clients."

Comment (pfulton) [reply build_process/README.md]: Yeah, I'm not sure this is the world's greatest sentence. This was existing content that I had left in here.

Now that I'm looking at this README with fresh eyes, I think I should take another pass at this, and remove (or radically edit) that Quality Expectations bit, but also include a "Deploy" section and (maybe) a "Delivery" section, as the headline at the very top of the document indicates.

What do you think?

Comment (nrleak) [reply build_process/README.md]: From what I see, I think that's a really good idea. I would just suggest running the idea by Adam really quick to make sure.

Comment (pfulton) [reply build_process/README.md]: All right, I've updated this to remove the Quality Expectations bit, but also created #205 to note that maybe that stuff should exist elsewhere? I also wrote up pieces on Deployment and Delivery

**Pull request merged**